### DISCUSSION TITLE
Conversion to TS

### DISCUSSION THREAD

**Discussion by fullStackDataSolutions** [timlrx/tailwind-nextjs-starter-blog#646] [Q&A]
How do I connect the blog to TypeScript?

**Comment by dlarroder** [comment]
What do you mean by TS? If you mean TypeScript, there seems to be no conversion guide for this project at the moment, but you can initialize the TypeScript version of this repo buy running:

```
npx degit 'timlrx/tailwind-nextjs-starter-blog#contentlayer'
```

**Reply by fullStackDataSolutions** [reply]
Thanks, doesn't seem to hard to transfer my stuff over to a new repo, so I'll just do this.
